api_docs/include/rest-endpoints.md

```diff
@@ -160,3 +160,4 @@
 * [Add an FCM registration token](/api/add-fcm-token)
 * [Remove an FCM registration token](/api/remove-fcm-token)
 * [Create BigBlueButton video call](/api/create-big-blue-button-video-call)
+* [Register E2EE push device](/api/register-push-device)
```

Comment:
I'd put this before the 4 device token endpoints.

Reply:
Done.